practices/jardondiego/pr3/pr3.ipynb

```diff
@@ -0,0 +1,669 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    # print([word for rank, word, *_ in dataset if rank &lt; 10])
</pre><p>No incluir código comentado. Mejor eliminarlo</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/umoqnier/cl-2024-2-lab/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>